tools/autograd/gen_variable_type.py

```diff
@@ -80,6 +80,28 @@
     '_coalesced_',
 }
 
+# When a function modifies its input tensors, it should never change the the input tensors'
```

Comment:
just to clarify a little bit "modifies its input tensors (via inplace or out-variants)..."